src/cmd/flux-job.c

```diff
@@ -108,6 +108,9 @@ static struct optparse_option attach_opts[] =  {
     { .name = "show-events", .key = 'E', .has_arg = 0,
       .usage = "Show job events on stderr",
     },
+    { .name = "show-exec", .key = 'G', .has_arg = 0,
```

Comment:
Not even sure you need a short opt here, but would -X make more mnemonic sense than -G?